server/metrics_server.go

```diff
@@ -0,0 +1,34 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/log"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var metricsServer *http.Server
+
+// StartMetricsServer starts a new HTTP server forthat exposes Kiali internal metrics in Prometheus format
+func StartMetricsServer() {
+	conf := config.Get()
+	log.Info("Starting Metrics Server")
```

Comment:
When logging this, indicate what the host and port is - this will be useful to help us know if a user configured these differently - we can know what they configured the metrics port to be, for example:

```suggestion
	log.Info("Starting Metrics Server on [%v:%v]", conf.Server.Address, conf.Server.MetricsPort)
```